### PR TITLE
Update free-programming-cheatsheets.md

### PR DESCRIPTION
**Pull request by Tnguyen-learning**
## What does this PR do?
Add resource(s) 

### Description
Add ggplot2 library  cheat sheet for R 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Tnguyen-learning** [comment]
add R cheat sheet

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -325,6 +325,7 @@
 ### R
 
 * [All RStudio cheatsheets resources](https://www.rstudio.com/resources/cheatsheets) - RStudio.com (HTML site with PDF links)
+* [Data Visualization with ggplot2 : : CHEAT SHEET](https://www.maths.usyd.edu.au/u/UG/SM/STAT3022/r/current/Misc/data-visualization-2.1.pdf) - maths.usyd.edu.au (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:328]: the credit for this should be http://ggplot2.tidyverse.org  Why is the link from ww.maths.usyd.edu.au ?

**Comment by Tnguyen-learning** [comment]
@eshellman Could you please review my work?

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by Tnguyen-learning** [comment]
@eshellman I have resolved the conflicts.

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -339,6 +339,7 @@
 * [All RStudio cheatsheets resources](https://www.rstudio.com/resources/cheatsheets) - RStudio.com (HTML site with PDF links)
 * [Base R Cheatsheet](https://iqss.github.io/dss-workshops/R/Rintro/base-r-cheat-sheet.pdf) - Mhairi McNeill (PDF)
 * [Cheat Sheet for R and RStudio](https://www.ocf.berkeley.edu/~janastas/docs/RStudioCheatSheet.pdf) - L. Jason Anastasopoulos (PDF)
+* [Colors in R](http://www.stat.columbia.edu/~tzheng/files/Rcolor.pdf) - Dr. Ying Wei  (PDF)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:342]:
```suggestion
* [Colors in R](http://www.stat.columbia.edu/~tzheng/files/Rcolor.pdf) - Ying Wei  (PDF)
```
We don't allow honorifications like `Dr.`

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Pull request merged**